src/lib/helpers/leave.ts

```diff
@@ -30,7 +30,7 @@ import {
 } from 'store/leave/common-screens/types';
 import {getDateFromString} from 'lib/helpers/time';
 
-const LEAVE_TYPE_COLORS = [
+export const LEAVE_TYPE_COLORS = [
   '#445abf',
```

Comment:
Better to add a comment as the color name (i.e 445abf - blue) for the ease of reference.